examples/self-supervised-learning/demo_equivariant_imaging.py

```diff
@@ -100,7 +100,7 @@
 # with a trainable denoising prior based on the DnCNN architecture.
 
 # Select the data fidelity term
-data_fidelity = dinv.optim.L2()
+data_fidelity = dinv.optim.data_fidelity.L2()
```

Comment:
is this necessary?

Reply:
No but I find it more clear, if it is clear that L2 is the L2 data-fidelity term and not the prior or the distance. It can be removed and it will work the same.